src/components/Buttons/SearchButton/SearchButton.tsx

```diff
@@ -1,12 +1,25 @@
 import React, { forwardRef } from 'react';
-import PropTypes from 'prop-types';
 import { FaSearch } from 'react-icons/fa';
 import { bem } from '../../../utils';
 import styles from './SearchButton.scss';
 
+export interface Props
+    extends Omit<
+        React.ButtonHTMLAttributes<HTMLButtonElement> &
+            React.AnchorHTMLAttributes<HTMLAnchorElement>,
+        'type'
+    > {
+    /** Used to disable the button */
+    disabled?: boolean;
+    /** Type of the button */
+    type: 'submit' | 'button';
```

Comment:
this used to be optional, and default to `button`. Let's keep it so.

Reply:
Fixed.